pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -548,6 +595,15 @@ func aclDropRules(aclArgs types.AppNetworkACLArgs) (types.IPTablesRuleList, erro
 		aclRule3.RuleID = 0xffffff
 		aclRule3.IsDefaultDrop = true
 		rulesList = append(rulesList, aclRule1, aclRule2, aclRule3)
+	case types.NetworkInstanceTypeSwitch:
+		aclRule2.Rule = append(aclRule2.Rule, "-m", "physdev",
```

Comment:
Is the intend to append to the aclRule2 which does the logging? If so a comment would be helpful.
What about aclRule1? If the intent is to ignore it it would be good to have a comment about it.

Reply:
Is there a reason we can't have the physdev match on aclRule2 for all cases?

Reply:
aclRule1 is for packets originating from app instance destined out to the internet. Yes, I think we have have the physdev-out match for aclRule2 in all cases.

Reply:
Will make the corresponding changes and add comments.

Reply:
I think we can have the physdev-out match rule for aclRule4 also for all network instance types.

Reply:
Actually I think we can put the physdev-in matches for aclRule1 and aclRule3 also for all cases.
I'll make the changes and test.